packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -77,6 +78,32 @@ export enum AuthenticationMode {
 
 const objectForGlobals = typeof globalThis != "undefined" ? globalThis : typeof window != "undefined" ? window : undefined;
 
+const liveQueryExchange: Exchange = ({ forward }) => {
```

Comment:
nit: can we move this into https://github.com/gadget-inc/js-clients/tree/connection-reexecute/packages/api-client-core/src/exchanges and add a comment explaining why it exists?